src/openrct2/scripting/bindings/object/ScObject.hpp

```diff
@@ -215,11 +215,6 @@ namespace OpenRCT2::Scripting
 
         uint8_t numVerticalFrames_get() const
         {
-            auto entry = GetEntry();
```

Comment:
@IntelOrca not sure what would be best for this? This field is unused and so is `numHorizontalFrames_get` so it would be cleanest to just remove both of these altogether.

Both would get replaced by `numRotationFrames_get` which unlike the current code wouldn't always return 0.